test

```diff
@@ -185,16 +188,12 @@ def test_all(year, event, problem, language, id):
 
   checker = prepare_checker(language, file)
 
-  results = [
-    test(checker, input, expected)
-    for input, expected in test_case
-  ]
+  for input, expected in test_case:
+    yield test(checker, input, expected)
```

Comment:
WTFFFFFF, why you are using a yield inside of a global scope of a `class`? I don't understand the reason for that, you can explain me, please?